### ISSUE TITLE
Does tc-lib-pdf support font kerning and tracking ?

### ISSUE THREAD

**Issue by solexsylvain** [tecnickcom/tc-lib-pdf#35]

**Comment by nicolaasuni** [comment]
Yes via the spacing and stretching font properties in https://github.com/tecnickcom/tc-lib-pdf-font
See: https://github.com/tecnickcom/tc-lib-pdf-font/blob/main/src/Stack.php#L155-L156
and the examples/index.php file in this library.
